api/config/initializers/cors.rb

```diff
@@ -8,7 +8,10 @@
 # Read more: https://github.com/cyu/rack-cors
 
 Rails.application.config.middleware.insert_before 0, Rack::Cors do
-  BASE_ORIGINS = ['https://new.hackclub.com', 'https://hackclub.com'].freeze
+  BASE_ORIGINS = ['https://new.hackclub.com',
+                  'http://new.hackclub.com',
```

Comment:
@MaxWofford: Interesting... It works for me when I go to `http://hackclub.com` in a private Chrome session.